8Knot/pages/welcome/welcome.py

```diff
@@ -13,56 +15,349 @@
 dash.register_page(__name__, path="/", order=1)
 
 
-layout = dbc.Container(
-    className="welcome_container",
+layout = html.Div(
+    style={
+        "background": "#FFFFFF",  # pure white like Figma base
+        "minHeight": "100vh",
+        "padding": "0",
+        "margin": "0",
+        "position": "relative",
+    },
     children=[
-        html.Div(
-            className="toplevel_welcome_div",
+        dbc.Container(
+            fluid=True,
+            style={"padding": "0"},
             children=[
+                # unified section with gradients through the page
                 html.Div(
-                    className="welcome_callout_section",
+                    style={
```

Comment:
Apply all these styling changes in the css file using the classname. See how the rest of the styling has been applied